modules/light-clients/08-wasm/types/client_state.go

```diff
@@ -96,18 +87,34 @@ func (cs ClientState) ZeroCustomFields() exported.ClientState {
 	return &cs
 }
 
+type (
+	timestampAtHeightInnerPayload struct {
+		Height exported.Height `json:"height"`
+	}
+	timestampAtHeightPayload struct {
+		TimestampAtHeight timestampAtHeightInnerPayload `json:"timestamp_at_height"`
+	}
+)
```

Comment:
this is also superseded by #4133, right?

I'd like to not use `type` blocks, e.g.:

```go
type (
    someType struct {}

    someOtherType struct {}
)
```

instead:

```go
type someType struct {}

type someOtherType struct {}
```

Reply:
Just for my own understanding: what's wrong with `type` blocks?

Reply:
@chatton doesn't like them! 😆 

I think I have similar feelings!

Reply:
Well, I don't want to get @chatton angry, so...